### PR TITLE
Task 3 login notif

### PR DESCRIPTION
**Pull request by DanielDaCosta**
# Description
This PR introduces reusable Success and Error Notification components, built upon Material-UI's SnackBar. The "Notification" component is designed to provide consistent and user-friendly feedback across the application.

## Type of Change
Please delete options that are not relevant:

[x] New feature (non-breaking change that adds functionality)

 **Objective:** Create toast notifications for both successful and failed login attempts.
 **Activities:** Design the UI using Material UI and integrate it with the NextJS and ReactJS frontend.

### CODE REVIEW

File: frontend/templates/SignIn/SignInForm/SignInForm.jsx [hunk 1]
```diff
@@ -153,12 +170,57 @@ const SignInForm = (props) => {
...
+          </p>
+          <p style={{ color: '#ffb3b3', margin: 0 }}>
+            This account is not registered to a Radical account <span style={{ color: '#ff6f6f', fontWeight: 'bold' }}>Create an account</span> to continue.
+          </p>
+        </div>
+        }
+        {...styles.errorNotificationProps}
+      />
+      // <SuccessNotification
+      //   open={openError}
```
Comment (rajmal-sk) [inline frontend/templates/SignIn/SignInForm/SignInForm.jsx]: Removed the commented code.

File: frontend/templates/SignIn/SignInForm/SignInForm.jsx [hunk 2]
```diff
@@ -153,12 +170,57 @@ const SignInForm = (props) => {
...
+
+  
+  const renderErrorNotification = () => {
+    return (
+      <ErrorNotification
+        open={openError}
+        onClose={handleClose}
+        message={
+          <div>
+          <p style={{ fontWeight: 'bold', color: '#ff6f6f', margin: 0, display: 'flex', alignItems: 'center' }}>
```
Comment (rajmal-sk) [inline frontend/templates/SignIn/SignInForm/SignInForm.jsx]: Construct message and refer it in error notification.

File: frontend/templates/SignIn/SignInForm/SignInForm.jsx [hunk 3]
```diff
@@ -99,7 +108,15 @@ const SignInForm = (props) => {
       dispatch(setLoading(true));
       router.push(ROUTES.HOME);
     } catch ({ code }) {
-      setError({ password: { message: AUTH_ERROR_MESSAGES[code] } });
+      console.log(code);
```
Comment (rajmal-sk) [inline frontend/templates/SignIn/SignInForm/SignInForm.jsx]: Remove console logging

File: frontend/components/Notification/styles.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+            background: '#E6DBFF',
+            boxShadow: '0px 10px 22.2px 0px #00000021',
+            fontSize: '16px',
+            fontFamily: 'Satoshi, sans-serif', // Using Satoshi for the main message
+        }
+    },
+    // successNotificationPosition: {
+    //     anchorOrigin: { vertical: 'top', horizontal: 'center' },
+    //     sx: {
+    //         position: 'absolute',
```
Comment (rajmal-sk) [inline frontend/components/Notification/styles.js]: Remove commented out code

File: frontend/components/Notification/styles.js [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+        anchorOrigin: { vertical: 'top', horizontal: 'center' },
+        sx: {
+            position: 'absolute',
+            top: '7%',
+            // height: '200px',
+            left: '50%',
+            transform: 'translate(-50%, -50%)',
+            width: '50%', // Adjust the max-width as needed
+
+            // maxWidth: '50%', // Adjust the max-width as needed
```
Comment (rajmal-sk) [inline frontend/components/Notification/styles.js]: Remove commented code

File: frontend/components/AuthForm/AuthForm.jsx [hunk 6]
```diff
@@ -34,6 +38,36 @@ const AuthForm = (props) => {
...
+  //       <Box
+  //         display="flex"
+  //         justifyContent="center"
+  //         alignItems="center"
+  //         height="100%"
+  //       >
+  //         {notification}
+  //         {/* <Snackbar
+  //           open={true}
+  //           autoHideDuration={6000}
```
Comment (rajmal-sk) [inline frontend/components/AuthForm/AuthForm.jsx]: Remove all the commented code

File: frontend/components/AuthForm/AuthForm.jsx [hunk 7]
```diff
@@ -4,11 +4,14 @@ import Link from 'next/link.js';
 
 import styles from './styles.js';
 
+// import { Snackbar, Button } from '@mui/material';
```
Comment (rajmal-sk) [inline frontend/components/AuthForm/AuthForm.jsx]: Remove unused imports

**Comment by DanielDaCosta** [comment]
All changes were made. @yhannineh can you please approve the review so that I can merge? @rajmal-sk

**Review by yhannineh — APPROVED** [review]

**Pull request merged**